nextcore/http/client/client.py

```diff
@@ -45,10 +45,12 @@
 from ..rate_limit_storage import RateLimitStorage
 from ..route import Route
 from .base_client import BaseHTTPClient
+from .client_response import ClientResponse
 
 if TYPE_CHECKING:
     from typing import Any, Final, Literal
 
+    # ClientResponse is still imported here because ours is "incompatible"
```

Comment:
This is incredibly cursed. Wouldnt it make more sense to make ours compatible?

Reply:
Pyright isn't being specific on how it's "incompatible", so I'm not sure how to make it compatible.

Reply:
>   <path>/nextcore/nextcore/http/client/client.py:316:28 - error: Expression of type "ClientResponse" cannot be assigned to return type "ClientResponse"
    "aiohttp.client_reqrep.ClientResponse" is incompatible with "nextcore.http.client.client_response.ClientResponse" (reportGeneralTypeIssues)

Here's what I'm getting, for reference.

Reply:
Weird. Will look into it eventually tm